tekton/rpmbuild/rpmbuild-run.yml

```diff
@@ -1,16 +1,20 @@
+
 ---
-apiVersion: tekton.dev/v1alpha1
+apiVersion: tekton.dev/v1beta1
 kind: TaskRun
 metadata:
   name: rpmbuild-taskrun
 spec:
   taskRef:
+    kind: Task
     name: rpmbuild
-  inputs:
-    resources:
+  resources:
+    inputs:
       - name: source
-        resourceRef:
-          name: tektoncd-cli-git
-    params:
-    - name: copr-cli-secret
-      value: copr-cli-config
+        resourceSpec:
+          type: git
+          params:
+            - name: revision
+              value: master
```

Comment:
Do we need the master revision or the release branch/tag

Reply:
yes, this is to checkout the file to build (ie build.sh) the rpm not for the release version.